source/create-agg-pipeline.txt

```diff
@@ -372,10 +371,18 @@ Learn More
 
 - :ref:`view-pipeline-explain-plans`
 
+- :ref:`compass-export-pipeline`
```

Comment:
[Suggestion]

Users can access most of these links from the left toc. What do you think about trimming down this list to the top 3-5 pages we think might the be most useful to direct users to?

Reply:
That makes sense! Can link out some "next steps" within the task pages.